src/backend/mod.rs

```diff
@@ -133,7 +133,10 @@ impl Backend {
         if dpi < 0.5 {
             bug!("unexpected dpi: {}", dpi);
         } else if dpi.fract().min(1.0 - dpi.fract()) > std::f64::EPSILON {
-            bug!("fractional HiDPI scaling is not yet supported: {}", dpi);
+            warn!(
+                "fractional HiDPI scaling is not yet supported. but may work! (dpi: {})",
```

Comment:
```suggestion
                "fractional HiDPI scaling is not yet fully supported! (dpi: {})",
```

Reply:
True

Reply:
Once you've changed this I am going to merge your PR :+1: I don't want to accept your PR and then adapt it later as that would be a waste of both of our time.

Thanks for your interest and readiness to help :heart: